smplx_blender/data/smplx_betas_to_joints_female.json

```diff
@@ -0,0 +1 @@
008833598345518112], [0.1130644753575325, -1.23379385471344, 0.06459929794073105], [-0.115655317902565, -1.2309560775756836, 0.06683986634016037], [-0.0060583376325666904, 0.11902894824743271, -0.0311281718313694], [0.05194227024912834, 0.03923298045992851, -0.00914478674530983], [-0.054993800818920135, 0.03520824387669563, -0.01564321108162403], [0.00271681253798306, 0.2651205062866211, -0.010384751483798027], [0.14042088389396667, 0.08478745818138123, -0.036786291748285294], [-0.15059725940227509, 0.08306162059307098, -0.03640877828001976], [0.41266506910324097, 0.028904646635055542, -0.06406506150960922], [-0.41315433382987976, 0.03228379786014557, -0.056708112359046936], [0.6400852203369141, 0.036446936428546906, -0.05998050794005394], [-0.6412692070007324, 0.03563779219985008, -0.061463624238967896], [-0.0003815271775238216, 0.26234760880470276, -0.017154766246676445], [0.030958179384469986, 0.29982680082321167, 0.05825266242027283], [-0.03095882572233677, 0.2998262941837311, 0.0582512728869915], [0.740061342716217, 0.021942008286714554, -0.04017968475818634], [0.770990788936615, 0.02466127835214138, -0.037521589547395706], [0.7928956747055054, 0.023296931758522987, -0.03712322562932968], [0.7468849420547485, 0.02426680363714695, -0.06222943589091301], [0.7762548923492432, 0.025630177929997444, -0.06603347510099411], [0.7990442514419556, 0.02508469671010971, -0.06971845775842667], [0.722144603729248, 0.016788890585303307, -0.09959980845451355], [0.7364270091056824, 0.016381343826651573, -0.11131137609481812], [0.7509890794754028, 0.015516691841185093, -0.1226668730378151], [0.7351051568984985, 0.021683238446712494, -0.08427481353282928], [0.7618342041969299, 0.02333959750831127, -0.08952222019433975], [0.7836958765983582, 0.022405020892620087, -0.09649993479251862], [0.6816923022270203, 0.014549827203154564, -0.03408398851752281], [0.6983776688575745, 0.015283027663826942, -0.011016267351806164], [0.7181630730628967, 0.009661861695349216, 0.002971678040921688], [-0.74006187915802, 0.021944520995020866, -0.040177930146455765], [-0.7709909081459045, 0.024662749841809273, -0.037520863115787506], [-0.792895495891571, 0.02329767867922783, -0.0371231734752655], [-0.7468854188919067, 0.02426842600107193, -0.06222955882549286], [-0.7762554287910461, 0.025631628930568695, -0.06603360176086426], [-0.7990444898605347, 0.025085289031267166, -0.06971906870603561], [-0.7221466302871704, 0.01678796112537384, -0.09960071742534637], [-0.7364281415939331, 0.016380809247493744, -0.11131169646978378], [-0.7509896755218506, 0.015516774728894234, -0.12266724556684494], [-0.7351065278053284, 0.02168373018503189, -0.08427593111991882], [-0.7618348002433777, 0.023340435698628426, -0.08952361345291138], [-0.7836961150169373, 0.022405201569199562, -0.09650153666734695], [-0.6816904544830322, 0.014549721963703632, -0.03408147022128105], [-0.6983757615089417, 0.015283329412341118, -0.011013735085725784], [-0.718163013458252, 0.009662178345024586, 0.002972340676933527]]}
```

Comment:
Should / can these be NPY files? Storing floats in JSON for use in calculations seems hinky from both a precision and size perspective. 

Similarly -- what's in the PKLs and can they be NPYs instead? Pickle as a storage format has issues.

Reply:
I'm fine with converting, I was just leaving previous functionality alone (only refactoring, not changing). I'll do so in a separate PR!